config/index.js

```diff
@@ -623,6 +623,12 @@ var conf = convict({
       format: Boolean,
       env: 'SMS_ENABLED'
     },
+    useMock: {
+      doc: 'Use a mock SMS provider implementation, for functional testing',
+      default: false,
+      format: Boolean,
+      env: 'SMS_USE_MOCK'
```

Comment:
I wondered that too, and don't have any good reasons why I chose this approach other than it was simple to implement.